app/src/main/java/com/tip/lunchbox/data/Repository.java

```diff
@@ -1,4 +1,32 @@
 package com.tip.lunchbox.data;
 
+import com.tip.lunchbox.model.DailyMenuResponse;
+import com.tip.lunchbox.model.Restaurant;
+import com.tip.lunchbox.model.Reviews;
+import com.tip.lunchbox.model.SearchResponse;
+
+import java.util.List;
+import java.util.Map;
+
+import io.reactivex.rxjava3.core.Observable;
+
 public class Repository {
+
+    public Observable<SearchResponse> getSearchResponseObservable(Map<String,String> QueryMap) {
+        return ApiManager.getApiService().getSearchResponse(QueryMap);
+    }
+
+    public Observable<DailyMenuResponse> getDailyMenuResponseObservable(int restaurantID) {
+        return ApiManager.getApiService().getDailyMenuResponse(restaurantID);
+    }
+
+    public Observable<List<Reviews>> getReviewsResponseObservable(int restaurantID ,
+                                                                  Integer start,Integer count) {
```

Comment:
@epicadk you will also need to reflect the changes from [APIService](https://github.com/devansh-299/LunchBox/pull/12#discussion_r481587011) in this method call also

Reply:
Done